app/src/main/java/com/powerhungers/fomezero/feature/login/data/remote/model/LoginResponse.kt

```diff
@@ -21,4 +21,8 @@ data class LoginResponse(
     val createdAt: String?,
     @SerializedName("updatedAt")
```

Comment:
```suggestion
    @SerializedName("updated_at")
```
os valores serializados dessa classe estão incorretos, o que acha de ja corrigir esses erros